rageval/metrics/_answer_rouge_correctness.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+
+from typing import List, Any, Callable, Optional
+from rouge_score import rouge_scorer
+import datasets
+
+from datasets import Dataset
+from dataclasses import dataclass
+
+from rageval.metrics import Metric, add_attribute
+
+_DESCRIPTION = """Estimates ROUGE score by estimating answer and groundtruth answers.
+
+ROUGE is case insensitive, so the input text is converted to lower case before computing the score. This metrics is a wrapper around the https://github.com/google-research/google-research/blob/master/rouge/rouge_scorer.py
+
+"""
+
+_KWARGS_DESCRIPTION = """\
+Args:
+    name : str
+    rouge_type : str, the rouge type to calculate. Defaults to 'rouge1', 'rouge2', 'rougeL', 'rougeLsum'
+        "rouge1": unigram (1-gram) based scoring
+        "rouge2": bigram (2-gram) based scoring
+        "rougeL": Longest common subsequence based scoring.
+        "rougeLSum": splits text using "\n".
+
+Optional Args:
+    tokenizer : Callable, a tokenizer can be passed to the scorer, replacing the default tokenizer which tokenizes on whitespace, especially for non-latin languages. For example, the `jieba.cut` can be used for Chinese.
+
+Functions:
+    _compute_one: compute the score by measure whether the args:`answer` contains short answer in list:`gt_answers`.
+
+Examples:
+    >>> from datasets import Dataset
+    >>> import rageval as rl
+    >>> sample = {"answers": ["test answer"],"gt_answers":[["test gt_answer", "test groundtruth answer"]]}
```

Comment:
replace the sample with a more realistic case.

Reply:
updated.